pkg/controller/zerocapacity/zero_controller.go

```diff
@@ -293,12 +293,10 @@ func (z *Controller) cleanupResources(httpClient http.HttpClient, request reconc
 	var allErrors error
 
 	// Stop the zero-capacity server so that it leaves the Infinispan cluster
-	rsp, err, reason := httpClient.Post(request.Name, consts.ServerHTTPServerStop, "", nil)
+	rsp, err, reason := httpClient.Post(meta.Name, consts.ServerHTTPServerStop, "", nil)
```

Comment:
@ryanemerson it looks like we have duplicate methods for shutdown server or it's temporary solution just for tests as we have `cluster.GracefulShutdown` method

Reply:
Graceful shuts down the cluster not the individual server.

Reply:
Oh, true, missed this.
In this case we should review some cluster vs individual server interaction and try to optimize it. In some cases Pod[0] can be very heated, but we can interact with cluster via service, as it's much more faster than call curl inside Pod.

Reply:
In this case we need to speak to a specific pod, the zero-capacity one, so I'm not sure we can go via the service. I could make a Server interface similar to `ClusterInterface` to abstract the details of the REST call for future reuse

Reply:
Yes, sure, this not for current case, just to discuss and remember it.